backend/api/Areas/Project/Models/Workflow/ProjectStatusModel.cs

```diff
@@ -20,6 +20,16 @@ public class ProjectStatusModel : Api.Models.CodeModel<int>
         /// get/set - whether or not this status is a milestone, requiring special transition logic.
         /// </summary>
         public bool IsMilestone { get; set; }
+
```

Comment:
This file looks exactly the same as https://github.com/bcgov/PIMS/pull/383/files#diff-fb4b4bc721a7b997ed31135d056133d2 down to the same name. What makes this model different to the other one?

Reply:
models are controller specific to ensure changes don't break other endpoints.  It has the negative side effect where we will have to update multiple models when we want a change to go everywhere.  However this is more desirable than the other.